e2e/atlas_clusters_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// +build e2e
+
+package e2e
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+)
+
+func TestAtlasClusters(t *testing.T) {
+	cliPath, err := filepath.Abs("../bin/mcli")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+	_, err = os.Stat(cliPath)
+
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
+
+	atlasEntity := "atlas"
+	clustersEntity := "clusters"
+	clusterName := fmt.Sprintf("e2e-cluster-%v", r.Uint32())
+
+	t.Run("Create", func(t *testing.T) {
```

Comment:
[q] Is it possible for us to define command keywords centrally somewhere, so they can be reused here and the cli definitions?